src/components/molecules/inputs/Input/Input.tsx

```diff
@@ -63,6 +63,16 @@ export const Input: Input = (props) => {
             {children}
           </TextInput>
         )
+
+      case 'number':
+        return (
+          <TextInput
+            {...props}
+            pattern={pattern || /^[0-9]{2,30}$/}
```

Comment:
is there a reason to limit the default pattern to length between 2 - 30 chars? seems so random to me

Reply:
This is limited due to the width-limitation of the input-field :D I dont know why exactly 30 but the limit was already set for another custom pattern with numbers (dont remember the exact code-line but somewhere in the storires )